build/ghactions/pr-codecov-run-tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+output_json_file="$1"
+packages="$2"
+
+if [ -z "${packages}" ]; then
+  echo "Skipping"
+  touch "${output_json_file}"
+  exit 0
+fi
+
+
+# Find the targets. We need to convert from, e.g.
```

Comment:
Nit: in `changed-go-pkgs`, we go to efforts to output the package name with the leading `./` (`echo -n "./$path "`), but here we struggle to convert if to something else. If you just output the names without the leading `./` in the first place then there's less back-and-forth conversion to do.